src/windowsdesktop/src/windowsdesktop/src/bundle/Wix.targets

```diff
@@ -0,0 +1,303 @@
+<Project>
+  <!-- Import VersionDetails.props for direct version property access -->
+  <!-- Extracted WiX build targets for Windows Desktop Runtime bundle -->
+  
+  <UsingTask TaskName="DownloadFile" AssemblyFile="$(ArcadeSdkBuildTasksAssembly)" />
+
+  <Target Name="CollectRuntimePrereqDependencies" Condition="'$(RestoreRuntimePrereqs)'!='false' AND '$(TargetArchitecture)'!=''">
+    <PropertyGroup>
+      <RuntimePrereqStageDirectory>$(ArtifactsRoot)prereqs\$(TargetArchitecture)\</RuntimePrereqStageDirectory>
+      <RuntimePrereqStageDirectory Condition="'$(ArtifactsRoot)'==''">$(MSBuildThisFileDirectory)..\..\..\..\artifacts\prereqs\$(TargetArchitecture)\</RuntimePrereqStageDirectory>
+      <RuntimePrereqDownloadDirectory>$(BaseIntermediateOutputPath)runtime-prereqs\$(TargetArchitecture)\</RuntimePrereqDownloadDirectory>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <RemoteAsset Include="$(MicrosoftNETCorePlatformsVersion)/dotnet-host-$(MicrosoftNETCoreAppRefVersion)-win-$(TargetArchitecture).msi">
```

Comment:
Product.targets and Wix.targets appears to essentially have the same code. They also seem to both download the runtime deps?

Reply:
https://github.com/dotnet/dotnet/pull/2849/files#diff-fb490ad1a8e549d66f6bff26779e3b50bc5f98ad877904020b3d222cb3b1304bR9-R84